frontend/pages/Landing.vue

```diff
@@ -5,12 +5,29 @@ import MeetupSpeakers from '@/components/Landing/MeetupSpeakers.vue'
 import AboutMeetup from '@/components/Landing/AboutMeetup.vue'
 import JoinUs from '@/components/Landing/JoinUs.vue'
 import Counters from '@/components/Landing/Counters.vue'
+import { onMounted, ref, computed } from 'vue'
+import { useFindNextMeetup } from '@/composables/useFindNextMeetup.js'
+
+const meetups = ref([])
+
+const findNextMeetup = computed(() => {
+  if (meetups.value.length == 0) return {}
+  const { nextMeetup } = useFindNextMeetup(meetups.value) 
+  return nextMeetup.value
+})
+
+onMounted(async() => {
+    await fetch('/api/meetups.json').then((response) => response.json()).then((data) => {
+    meetups.value = data
+  })
```

Comment:
```suggestion
  await fetch('/api/meetups.json').then((response) => response.json()).then((data) => {
    meetups.value = data
  })
```
Also: I suggest to read something like this (https://dmitripavlutin.com/javascript-fetch-async-await/) to understand how await/async and fetch/then works. I would prefer to use one of these approaches.